libs/contrib/Data/Always.idr

```diff
@@ -0,0 +1,39 @@
+module Data.Always
+
+import Data.Elem
+
+%default total
+
+data Always : (x -> Type) -> List x -> Type where
```

Comment:
`x` is a bit of an unconventional name for a type variable here - typically, we use `a`, `b`, `c` or something descriptive like `elem` to range over types.

Also, I'd expect the name `Always` to refer to the similar predicate over streams, rather than the List version.